packages/admin/admin/src/common/buttons/split/SplitButton.tsx

```diff
@@ -1,36 +1,95 @@
 import { ChevronDown } from "@comet/admin-icons";
-import { Button, ButtonGroup, ButtonGroupProps, MenuItem, MenuList, Popover, PopoverProps } from "@mui/material";
-import { withStyles } from "@mui/styles";
+import {
+    Button,
+    ButtonGroup as MuiButtonGroup,
+    ButtonGroupProps,
+    MenuItem as MuiMenuItem,
+    MenuList as MuiMenuList,
+    Popover as MuiPopover,
+} from "@mui/material";
+import { styled, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 import { PropsWithChildren } from "react";
 
 import { useStoredState } from "../../../hooks/useStoredState";
 import { SplitButtonContext } from "./SplitButtonContext";
 
-export interface SplitButtonProps extends ButtonGroupProps<any> {
+export type SplitButtonClassKey = "root" | "activeButton" | "popover" | "menuList" | "menuItem";
+
+export interface SplitButtonProps
+    extends ButtonGroupProps<any>,
+        ThemedComponentBaseProps<{
+            root: typeof MuiButtonGroup;
+            activeButton: typeof Button;
+            popover: typeof MuiPopover;
+            menuList: typeof MuiMenuList;
+            menuItem: typeof MuiMenuItem;
+        }> {
     selectIcon?: React.ReactNode;
     selectedIndex?: number;
     onSelectIndex?: (index: number, item: React.ReactElement) => void;
     showSelectButton?: boolean;
     localStorageKey?: string;
     autoClickOnSelect?: boolean;
     storage?: Storage;
-    popoverProps?: Partial<PopoverProps>;
```

Comment:
Removing this prop is a breaking change. 
Please either add a changeset mentioning `slotProps` should be used instead or keep the prop and mark it as deprecated.

Reply:
OK, I marked it as deprecated

Reply:
Please add the `popoverProps` back to the `Popover` then, in addition to `...slotProps?.popover` so it actually still works.